box-ui/src/components/ts/FormJoinMeeting.tsx

```diff
@@ -0,0 +1,108 @@
+import React, { FunctionComponent, useState } from 'react';
+import TextField from '@mui/material/TextField';
+import InputAdornment from '@mui/material/InputAdornment';
+import { Box, Button } from '@mui/material';
+import Keyboard from 'react-simple-keyboard';
+import 'react-simple-keyboard/build/css/index.css';
+import InputRoom from './JoinMeetingComponent';
+import '../css/FormJoinMeeting.css';
+interface InputRoom {
+    domain: string;
+    roomName: string;
+}
+interface FormJoinMeetingProps {
+    setInformation: (value: { domain: string; roomName: string }) => void;
+    close: () => void;
+    information: InputRoom;
+}
+const REGEX = new RegExp(/^([a-z0-9]+(-[a-z0-9]+)*\.)+[a-z]{2,}$/);
+
+const FormJoinMeeting: FunctionComponent<FormJoinMeetingProps> = (props: FormJoinMeetingProps) => {
+    const [domain, setDomain] = useState<string>(props.information.domain);
+    const [roomName, setRoomName] = useState<string>('');
```

Comment:
pourquoi initialiser avec un string vide et pas avec le props ?
```suggestion
    const [roomName, setRoomName] = useState<string>(props.information.roomName);
```

Reply:
Ok i am dumb